website/docs/r/workers_kv.html.markdown

```diff
@@ -35,10 +35,10 @@ The following arguments are supported:
 
 ## Import
 
-Workers KV Namespace settings can be imported using it's ID.  **Note:** Because the same key can exist in multiple namespaces, the ID is a composite key of the format <namespace_id>_<key>.
+Workers KV Namespace settings can be imported using it's ID.  **Note:** Because the same key can exist in multiple namespaces, the ID is a composite key of the format <namespace_id>/<key>.
 
 ```
-$ terraform import cloudflare_workers_kv_namespace.example beaeb6716c9443eaa4deef11763ccca6_test-key
+$ terraform import cloudflare_workers_kv.example beaeb6716c9443eaa4deef11763ccca6/"test-key"
```

Comment:
we don't need the quotes around the last bit here.

Reply:
That's likely a shell issue. The importer splits on `/` and spaces are fine. I'm not in front of a computer at the moment but I think `terraform import resource.foo "abc/key here"` will also work as expected.